tests/Moq.Tests/ProtectedAsMockFixture.cs

```diff
@@ -384,12 +418,51 @@ public void VerifyGet_includes_failure_message_in_exception()
 		public interface INested
 		{
 			int Value { get; set; }
+			int Method(int value);
 		}
+
 		public abstract class Foo
 		{
 			protected Foo()
 			{
 			}
+			private int _virtualSet;
+			public virtual int VirtualSet
+			{
+				get
+				{
+					return _virtualSet;
+				}
+				protected set
+				{
+					_virtualSet = value;
+				}
+
+			}
+
+			public void SetVirtual(int value)
+			{
+				VirtualSet = value;
+			}
+
+			private int _virtualGet;
+			public virtual int VirtualGet
+			{
+				protected get
+				{
+					return _virtualGet;
+				}
+				set
+				{
+					_virtualGet = value;
+				}
+
+			}
+
+			public int GetVirtual()
+			{
+				return VirtualGet;
+			}
```

Comment:
&dagger;) Is this part of the bugfix mentioned in the changelog? If so, please re-submit as part of a separate bugfix PR.